go-controller/pkg/ovn/base_network_controller_multipolicy.go

```diff
@@ -11,7 +11,10 @@ import (
 	"k8s.io/klog/v2"
 )
 
-const PolicyForAnnotation = "k8s.v1.cni.cncf.io/policy-for"
+const (
+	PolicyForAnnotation = "k8s.v1.cni.cncf.io/policy-for"
+	MultiNetPolicyLabel = "multinet-policy"
```

Comment:
is this a new label we expect users to add? 
I am totally spaced out on this design/discussion part totally